### PR TITLE
Pnr/bag count

### PR DESCRIPTION
**Pull request by kasimhelil**

### CODE REVIEW

File: src/pages/paxDetail/pnr/PNR.js [hunk 1]
```diff
@@ -5,13 +5,53 @@ import CardWithTable from "../../../components/cardWithTable/CardWithTable";
...
   const segmentTitle = `${data.recordLocator} ${
     hasData(data.version) ? `(Version: ${data.version})` : ""
   }`;
 
+  const getPassengerName = paxId => {
+    const passengers = data.passengers;
+    const passenger = passengers.find(passenger => passenger.paxId === paxId.toString());
+    return passenger;
+  };
+  const getBagData = data => {
```
Comment (gtasautumn) [inline src/pages/paxDetail/pnr/PNR.js:21]: We need some grouping here by flight and passenger to match the display of the original. Each passenger should have one row with a count of the bags, list of bag ids, etc.
bags_orig

This grouping is cleaner than having the passenger and total bag count repeated for each bag.

**Review by gtasautumn — CHANGES_REQUESTED** [review]
pls add the passenger grouping

File: src/pages/paxDetail/pnr/PNR.js (hunk 1 above)
Comment (kasimhelil) [reply src/pages/paxDetail/pnr/PNR.js:21]: - I didn't see the group by flight on my local; time to change my data!!!
- The reason I did't concatenate the bagIds was that the row would look clean. I will change that, Thanks!

Comment (gtasautumn) [reply src/pages/paxDetail/pnr/PNR.js:21]: no worries. I saw the same thing and checked in a little pr to give the cards more space.

**Review by gtasautumn — CHANGES_REQUESTED** [review]
Can you double-check the summation under bag count and total bag weight? The bag count sum at the flight level is greater than the  the numbers for the individual records. Also the total bag weight doesn't match what is displayed in the 8080 page.

**Comment by kasimhelil** [comment]
> Can you double-check the summation under bag count and total bag weight? The bag count sum at the flight level is greater than the the numbers for the individual records. Also the total bag weight doesn't match what is displayed in the 8080 page.

- I am a little confused with the count that the backend is returning. Each bag has a bag count, and I thought it was the total bag count associated with a passenger in a single flight or booking details. So, if I see 3 bagIds per passenger, the count should be 3. However,  this is not what I am getting. Probably, I am using a bad data. 

And the total bag weight on 8080 site is wrong, I think.

**Comment by gtasautumn** [comment]
I think the issue with the count is that it's returning you the sum on each row, so you don't need to aggregate it there. The bag weight is correct on the old site according to the database. Looks like some of hte numbers are being multiplied by the number of bags.
pnrbagsnew
pnrbagsold

**Pull request merged**